### PR TITLE
TP8

### PR DESCRIPTION
**Pull request by mihaicostin34**
### Pull Request Overview

This pull request adds tp8

### Testing Strategy

N/A

### TODO or Help Wanted

This pull request still needs a few more code output examples 

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Mihai Costin <mihaicostin342@gmail.com>

### CODE REVIEW

File: docs/tp/08/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+### Instructions logiques
+
+Pour tout opération logique, le résultat se trouvera toujours au sommet de la pile: 1 pour `VRAI`, 0 pour `FAUX`. Toutes les instructions prennent les deux premières valeurs de la pile et font la comparaison associée.
+
+| Instruction  | Opérandes | Description |
+|:-----------:|-----------|-------------|
+|`gt`, `ge`|- | Moins que/Moins que ou égal | 
+|`lt`, `le`|- | Moins que/Moins que ou égal | 
+|`eq`, `ne`|- | Égal/Différent| 
+
```
Comment (CAmi307) [inline docs/tp/08/index.md:46]: For mahematic instructions you specified the fact that the operations pops the first two values in the stack directly in the table and for each line, and for the logical instructions you specified it once before the table. Let's do it in the same way for both tables.

File: docs/tp/08/index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+On obtiendra le code:
+
+```
+push x
+push 4
+push 2
+call sum, 3
+```
+
+:::
```
Comment (CAmi307) [inline docs/tp/08/index.md:77]: Maybe we can give a similar example with the order in which the numbers should be pushed for non-commutative arithmetic operations

File: docs/tp/08/index.md (hunk 1 above)
Comment (mihaicostin34) [reply docs/tp/08/index.md]: Solved

File: docs/tp/08/index.md (hunk 2 above)
Comment (mihaicostin34) [reply docs/tp/08/index.md:77]: There is an example given for an expression containing division and the order of the operands can be seen. Should I also write an explanation in words or is the example enough?

Comment (CAmi307) [reply docs/tp/08/index.md:77]: No. That's fine. You're right

**Review by CAmi307 — APPROVED** [review]

**Pull request merged**